docs/install.md

```diff
@@ -0,0 +1,82 @@
+# Installing and configuration
+
+## Installation
+
+The pulp-cli package can be installed from a variety of sources. After installing, see the next
+section on how to configure pulp-cli.
+
+### From PyPI
+
+```
+pip install pulp-cli[pygments]  # colorized output
+pip install pulp-cli  # no color output
+```
+
+### From a source checkout
+
+```
+git clone <your_fork_url>
+cd pulp-cli
+pip install -e .
+```
+
+## Configuration
+
+The CLI can be configured by using a toml file.
+By default the location of this file is `~/.config/pulp/settings.toml`.
+However, this can be customized by using the `--config` option.
+Any settings supplied as options to a command will override these settings.
+
+Example file:
+
+```
+[cli]
+base_url = "https://pulp.dev"
+verify_ssl = false
+format = "json"
+```
+
+### Generate
+
+To generate a new configuration:
+
+```
+pulp config create --base_url "http://localhost"
+pulp config create -e  # with an editor (e.g. vim)
+pulp config create -i  # interactive prompts
+```
+
+### Validate
+
+To validate an existing config:
+
+```
+pulp config validate
+pulp config validate --strict  # validates that all settings are present
+```
```

Comment:
We could add a note that connectivity can be checked with `pulp status`, and may be that validation here is _only_ static analysis (is there a user friendly term for this?).